### ISSUE TITLE
objectID in nested field raises when using aggregate_polars_all

### ISSUE THREAD

**Issue by sibbiii** [mongodb-labs/mongo-arrow#219]
Hi,

I really appreciate the support for polars, but

```
collection.insert_one({'obj': {'data_to_test': bson.ObjectId()}})
pymongoarrow.api.aggregate_polars_all(collection, [], 
                                      schema=pymongoarrow.api.Schema({'obj': {'data_to_test': bson.ObjectId}}))
```

raises with `polars.exceptions.ComputeError: cannot create series from Extension("pymongoarrow.objectid", FixedSizeBinary(12), Some(""))` as the casting of FixedSizeBinary is not done for **nested** fields. 

_Ps.: For not nested fields it works fine. Loading the arrow data frame also works fine. So this is not a showstopper, but it prohibits from using the aggregate_polars_all convenience function._

See #220 for a fix

**Comment by ShaneHarvey** [comment]
Should this be closed now that #220 is merged?

**Comment by lazargugleta** [comment]
@ShaneHarvey yes
